app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,7 @@
   <%= csrf_meta_tags %>
 </head>
 <body class="<%= body_class %>">
+  <%= render "session_bar" -%>
```

Comment:
good to know! I looked up what it did when Responders prompted it, but didn't make sure it hadn't been deprecated.